timezone/src/include/timezone/dialogs.rb

```diff
@@ -316,7 +317,7 @@ def SetTimeDialog
                   HSpacing(3),
                   VBox(
                     Left(timeterm),
-                    VSpacing(),
+                    UI.TextMode ? Empty() : VSpacing(),
```

Comment:
Why not `VSpacing(UI.TextMode ? 0 : 1)`? That does not work here?

Reply:
Just was not sure about VSpacing default, so to left VSpacing() instead of forcing a `1`